pkg/topology/topology.go

```diff
@@ -104,7 +105,11 @@ func getDevicesFromVirtual(realDevPath string) (devs []string, err error) {
 func getTopologyHint(sysFSPath string) (*Hint, error) {
 	log.Debugf("getting topology hint for %s", sysFSPath)
 
-	hint := Hint{Provider: sysFSPath}
+	plainPath := sysFSPath
+	if sysRoot != "" {
+		plainPath = path.Join("/", strings.TrimPrefix(plainPath, sysRoot))
```

Comment:
`path.Join()` gets only single string argument to join with "/". No need to call it?

Reply:
Hmm. Dosen't `strings.TrimPrefix()` remove the leading '/' when we trim `sysRoot` ?

Reply:
I think it's good practice to use `Join()` as it's Clean()s the resulting path. A nano-scale philosophical nit that maybe it would be more correct to use `filepath.Join`

Reply:
Updated.

Reply:
My nit was really meaningless. I've also been bothered (and inconsistent) by that "duality" myself